src/widget/woverview.cpp

```diff
@@ -107,12 +107,18 @@ void WOverview::setup(const QDomNode& node, const SkinContext& context) {
         m_labelTextColor = Qt::white;
     }
 
-    bool okay = false;
-    m_iLabelFontSize = context.selectInt(node, "LabelFontSize", &okay);
-    if (!okay) {
-        m_iLabelFontSize = 11;
+    int labelFontSize = context.selectDouble(node, "LabelFontSize");
+    if (!labelFontSize) {
+        labelFontSize = 12.0;
     }
 
+    // Since we use the system font here, we need to adjust for full height
+    // of "Üg" and not just high of "Ug" set by setPixelSize
+    QFont refFont;
+    refFont.setPixelSize(labelFontSize);
+    QFontMetricsF fontMetrics(refFont);
+    m_labelFontSize = labelFontSize / fontMetrics.height() * labelFontSize;
```

Comment:
I don't understand this math. Why not just use `fontMetrics.height()`?

Reply:
The labelFontSize is the high of "Hg" decent() + capitalHight() 
My system font is Ubuntu, you have a different font. The ascent() is different.
Here we adjust the m_labelFontSize for an almost equal font independent hight().

I think I need to improve the comment.

Does this produce a good result on you side?

Reply:
>  almost equal font independent hight()

This goal does not make sense to me. The actual height of the text as it is rendered is what is needed.

> My system font is Ubuntu, you have a different font.

This seems to be the root of the issues. Does Shade not specify a font? If so, let's fix that issue so we get consistent results on both our computers.

Reply:
>> almost equal font independent hight()

> This goal does not make sense to me. The actual height of the text as it is rendered is what is needed.

Yes, but the actual height is finally kind of random since the skin designer does not know the system font. The skin designer can optimize the looking with the pixel size of its local font. If user uses a font with the same pixel size but taller or lower characters, the labels might overlap. This has happened on your and my system. This math here takes care, that the actual pixel size for the tall font is lower in a way that Üg = _Üg_ and not only Ug = _Ug_

Reply:
> Does Shade not specify a font?

It does: ```font-family: "Open Sans";```
But as long as we do not install the font it can be any fallback.

Reply:
But does that style get applied to WOverview? Does it need to be explicitly specified for WOverview?

Reply:
> the actual height is finally kind of random since the skin designer does not know the system font

This is the root problem that needs to be addressed. IMO it makes little sense to try to work around this with overcomplicated code.

Reply:
Ok, do you have an alternative solution?

Reply:
How about setting the maximum label height in the skin xml rather then setting the font pixel size?
This way the font can defined by style sheets and if it exceeds the lable the size is adjusted.

Reply:
I have tried replacing `font-family: "Open Sans"` with `font-family: Ubuntu` in Shade's style.qss and the label still gets hidden when hovering.

Reply:
The font used in my case is already Ubuntu. We need to find a way to define the Mixxx shipped font on all systems.

Reply:
With `font-family: Ubuntu`, I am able to increase the LabelFontSize for Shade up to 11. It is the same with "Open Sans". Can we just make the default font size 11?

Reply:
Ping. You can either explicitly specify the Ubuntu font or change the font size to 11. I think explicitly specifying the font would be better to ensure a consistent appearance.